package.json

```diff
@@ -28,11 +28,11 @@
   "devDependencies": {
     "@types/node": "^8.0.30",
     "ts-node": "^3.3.0",
-    "tslint": "5.7.0",
+    "tslint": "5.8.0",
```

Comment:
did you check what changed between versions?

Reply:
Yes, I did check. Primarily:
- new features/rules are added which we can discuss if we want to enable them if they are not enabled by default. Maybe separate PR and topic for discussion.
- Some rules were fixed for the bugs and some were enhanced. Was not possible to gauge the impact on our code via quick glance over the list. And while running `tslint` with the new version,  I just made sure that it doesn't throw up any new warnings or errors. 
- Certain options were deprecated on CLI which they plan to remove in further releases.